src/components/Search/SearchFilter.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '../Common/Button';
+
+const SearchFilter = (props) => {

```

Comment:
You can apply the same rule here to just return the component in your arrow function.